scripts/tests/test_SS.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import demographics
+import SS
+
+
+def test_get_pop_objs():
+    """
+    Test of the that omega_SS and the last period of omega_path_S are
+    close to each other.
+    """
+    E = 20
+    S = 80
+    T = int(round(4.0 * S))
+    start_year = 2018
+
+    (omega, g_n_ss, omega_SS, surv_rate, rho, g_n_vector, imm_rates,
+        omega_S_preTP) = demographics.get_pop_objs(E, S, T, 1, 100,
+                                                   start_year, False)
+    return omega_SS,imm_rates
+
+omega_SS,imm_rates = test_get_pop_objs()
+
+def test_SS():
+    """
+    Test whether the SS funtion works well and what the returns look like
+    """
+    r_init = 0.1
+    beta = 0.8
+    sigma = 1.5
+    alpha = 0.3
+    A = 1.0
+    delta = 0.1
+    xi = 0.1
+    S = 80
```

Comment:
I would recommend thinking of 3 periods as we did in class so you can compare the results with SS in that

Reply:
That's right - the answer will be different with exogenous labor.

Maybe you could have your scripts (e.g., SS.py) raise an assertion if the SS is not obtained before the maximum iterations are reached.  In that way, this test will fail if the SS isn't found and pass if it is.

Reply:
@jdebacker Yes. OK. Milton n I are working on that